### ISSUE TITLE
why Client apk download .update.apk and silent install in background?

### ISSUE THREAD

**Issue by linggarseptian** [EVLF/Cypher-Rat-Source-Code#35]
did you install another trojan silently?
found /mnt/sdcard/Download/.update.apk downloading from ready.apk after running on device

**Comment by kk159963** [comment]
yes I also found that he seems to have added other Trojans to the tool

seems to want to get user data

**Comment by EVLF** [comment]
really bro ?
cypher rat have feature named "Download apk" and here were the apk get download  /mnt/sdcard/Download/.update.apk
trust me the rat 100% clean on both side apk and controller , iam not that kind of people bro

**Comment by Frido-Cpp** [comment]
So what is the .update.apk for?

**Comment by 0-x27** [comment]
> really bro ? cypher rat have feature named "Download apk" and here were the apk get download /mnt/sdcard/Download/.update.apk trust me the rat 100% clean on both side apk and controller , iam not that kind of people bro

bro said to trust him LMFAOOO

**Comment by Mereid** [comment]
Did he actually install a trojan in background
